src/store/proofs.js

```diff
@@ -0,0 +1,18 @@
+import { defineStore } from 'pinia';
```

Comment:
Cool to start storing data in the store ! For instance #141

Reply:
But why in a seperate store ? Also, these are user proofs so could be under the user object, we could want to persist, and it avoid importing multiple stores, no ?

(sure in big apps they probably split stuff, but we don't need to introduce this complexity this soon, do we ?)

Reply:
The idea of this PR was also to start a discussion on separating stores, what should really be separated and how much data should be stored (if we do prices for example, there might be too much data).
I don't know the best practice, one thought was to have some kind of matching with backend models, so separate stores for User, Proof, Prices, Location, Stats...

Reply:
we can start the discussion once our store feels too big or complex ^^
I think we're far from that situation

Reply:
Cleaned up to use the unique store. Should be ready to merge